compiler/src/dotty/tools/dotc/transform/InstrumentCoverage.scala

```diff
@@ -97,7 +97,9 @@ class InstrumentCoverage extends MacroTransform with IdentityDenotTransformer:
         id = id,
         start = pos.start,
         end = pos.end,
-        line = pos.line,
+        // +1 to account for the line number starting at 1
+        // the internal line number is 0-base https://github.com/lampepfl/dotty/blob/18ada516a85532524a39a962b2ddecb243c65376/compiler/src/dotty/tools/dotc/util/SourceFile.scala#L173-L176
```

Comment:
We should also add parameter documention in `Statement`. 

https://github.com/lampepfl/dotty/blob/main/compiler/src/dotty/tools/dotc/coverage/Coverage.scala#L14-L26

Reply:
Thanks! do you mean, the parameter document that says `line` in `Statement` is 1-base ?

Reply:
I mean add something like this

```scala
/** A statement that can be invoked, and thus counted as "covered" by code coverage tools. 
 *
 *  @param location ...
 *  ...
 *  @param line 1-indexed line number
 *  ...
 */
case class Statement(
    location: Location,
    id: Int,
    start: Int,
    end: Int,
    line: Int,
    ...
```